Vagrantfile.fedora

```diff
@@ -23,7 +23,7 @@ Vagrant.configure("2") do |config|
       cat << EOF | dnf -y --exclude=kernel,kernel-core shell && break
 config install_weak_deps false
 update
-install iptables gcc make golang-go glibc-static libseccomp-devel bats jq git-core criu fuse-sshfs
+install iptables gcc golang-go make glibc-static libseccomp-devel bats jq git-core criu fuse-sshfs
```

Comment:
LOL, perhaps we should sort these alphabetically (assuming order doesn't matter here). At least that way we have something predictable.

Reply:
Yeah, I gathered it was a backport, and mostly a nit / reminder to self on my side 😅

Reply:
OK, instead of backporting #3442 and maybe #3670, I just changed the version.